client/stylesheets/_toCat.scss

```diff
@@ -4659,6 +4668,7 @@ On click the images will be displayed at normal size to complete the effect
 .searchResultsCardTitle a h2 {
     margin: 0 !important;
     color: #00bfff;
+    font-weight: 400 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces